### PR TITLE
BUG: Error message contained wrong values

### PR DESCRIPTION
**Pull request by TomAugspurger**
Closes https://github.com/scipy/scipy/issues/5814

I included a test, but I'm not sure how useful it is. Can easily remove it if need be.

EDIT: You've dropped support for 2.6 right? I think my generator expression would fail on that.

### CODE REVIEW

**Comment by larsmans** [comment]
2.6 is still supported according to `INSTALL.rst.txt`, but is no longer tested on Travis.

File: scipy/stats/stats.py [hunk 1]
```diff
@@ -228,9 +228,10 @@ def _chk2_asarray(a, b, axis):
 
 
 def _contains_nan(a, nan_policy='propagate'):
-    if nan_policy not in ('propagate', 'raise', 'omit'):
-        raise ValueError("nan_policy must be either 'propagate', 'raise', or "
-                         "'ignore'")
+    policies = ['propagate', 'raise', 'omit']
+    if nan_policy not in policies:
+        raise ValueError("nan_policy must be one of {%s}" %
+                         ', '.join("'%s'" % s for s in policies))
```
Comment (larsmans) [inline scipy/stats/stats.py]: Is this what you're worried about? This should work fine on Py2.6.

**Comment by rgommers** [comment]
> 2.6 is still supported according to INSTALL.rst.txt, but is no longer tested on Travis.

That's a mistake, the install docs should have been updated in gh-5756. I'll send a PR.

**Comment by larsmans** [comment]
Merging. The test is greatly appreciated, as this is obviously something that went wrong in the past. Thanks @TomAugspurger!

**Pull request merged**